pkg/rungroup/rungroup.go

```diff
@@ -76,6 +76,8 @@ func (g *Group) Run() error {
 		level.Debug(g.logger).Log("msg", "successfully interrupted actor", "actor", e.errorSourceName, "index", i)
 	}
 
+	level.Debug(g.logger).Log("msg", "done shutting own actors", "actor_count", len(g.actors), "initial_err", initialActorErr.err)
```

Comment:
```suggestion
	level.Debug(g.logger).Log("msg", "done shutting down actors", "actor_count", len(g.actors), "initial_err", initialActorErr.err)
```

?

Reply:
Already fixed this. And moved it to a defer